Dockerfile

```diff
@@ -1,16 +1,23 @@
+#FROM ubuntu:14.04
+#RUN  apt-get update \
+#  && apt-get install -y wget \
+#  && rm -rf /var/lib/apt/lists/*
 FROM public.ecr.aws/amazonlinux/amazonlinux:2022
 
 RUN yum -y update
+RUN yum install -y wget
 RUN yum install -y tar net-tools curl vim unzip less libevent-devel openssl-devel python-devel libtool git patch make gcc --allowerasing
-RUN git clone https://github.com/pgbouncer/pgbouncer.git --branch "pgbouncer_1_15_0" && \
+RUN wget https://github.com/jgm/pandoc/releases/download/3.1.6.2/pandoc-3.1.6.2-linux-amd64.tar.gz && \
```

Comment:
is there a cpu-platform agnostic option? i.e. no x86 or amd64 notation. I want us to think about multi-arch image that supports graviton and x86